packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -57,6 +58,12 @@ - (instancetype)initWithURL:(NSURL *)url
 static void *playbackBufferEmptyContext = &playbackBufferEmptyContext;
 static void *playbackBufferFullContext = &playbackBufferFullContext;
 
+#if TARGET_OS_IOS
+API_AVAILABLE(ios(9.0))
```

Comment:
iirc the minimum is already 9.0 (or above)?

Reply:
Yes, it's 9 on stable.